src/stories/components/MainMenu/styles.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Main Menu
+
+// .main-menu {}
+
+.menu-item {
+    min-width: 200px;
+    max-width: 200px;
+	padding: 12px 0 12px 35px;
+	display: flex;
+	align-items: center;
+    cursor: pointer;
+    font-weight:normal!important; 
+    margin-top: 15px;
+    margin-bottom: 15px;
+}
+
+.menu-item:hover {
+    background: linear-gradient(to right, $teal-dark 35%, white 42%);
+    fill: white;
```

Comment:
I think you can separate the icon as separate className, so you can style only the icon using something like  below for hover and active:
        border-radius: 0px 50px 50px 0px;
        background: $teal-dark;

Reply:
Also, the icons are not lined up properly with the text, the icons appear just a tad higher than the text. Once they are separated into their own class, you can also adjust the position to line up perfectly with the text.

Reply:
Thank you for advice. Fixed!